packages/website/src/components/PageContent.tsx

```diff
@@ -66,9 +66,9 @@ export const PageContent: FC<{ md: string; className?: string }> = ({
       components={{
         a(props) {
           // eslint-disable-next-line react/prop-types
-          if (props.href == null) {
+          if (props.href == null && props.name != null) {
             // eslint-disable-next-line react/prop-types
-            return <a href={props.href}>{props.children}</a>;
+            return <a id={props.name}>{props.children}</a>;
```

Comment:
@dtolb I wasn't sure why you were passing a null href through. Is there something here I was missing?

🙏

Reply:
@michaelglass how are regular links supposed to work here? Or any link for that matter, without an href attribute. You're probably right about the regression, I'm afraid we only tested them for regular markdown links from elsewhere on the page (not the generated docs).

Reply:
Oh never mind I understand now...